app/processing/delta/assign-ledger/get-invoice-line-total.js

```diff
@@ -0,0 +1,7 @@
+const getInvoiceLineTotal = (paymentRequest) => {
```

Comment:
I think that might be more complicated as we'd essentially just be putting a wrapper around `reduce`.  Maybe rather than that we consider a function that calculates the value of a payment request and the lines as that is a common use case?

Something to think about for further improvements.  I think I've exhausted the scope creep allowed for this ticket!